src/Translator.php

```diff
@@ -85,10 +91,6 @@ private function checkEnv(): void
         } catch (ProcessFailedException $e) {
             throw new RuntimeException('Could not find wget command. Please install it.');
         }
-        $this->txSite = getenv('TX_SITE');
-        if (!$this->txSite) {
-            throw new RuntimeException('TX_SITE environment variable is not defined');
-        }
         if (strpos($this->txSite, 'http://') !== 0 && strpos($this->txSite, 'https://') !== 0) {
             $this->txSite = 'http://' . $this->txSite;
         }
```

Comment:
This `txSite` validation needs to be moved to after `txSite` is assigned.

Reply:
Updated